actions/soak_helpers.go

```diff
@@ -5,18 +5,26 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 
+	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/geth"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
+	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/remotetestrunner"
 	"github.com/smartcontractkit/chainlink-testing-framework/config"
 	"github.com/smartcontractkit/chainlink-testing-framework/utils"
-	"github.com/smartcontractkit/helmenv/environment"
 )
 
 // Soak Test helpers
 
 // BuildGoTests builds the go tests to run, and returns a path to it, along with remote config options
-func BuildGoTests(executablePath, testsPath string) (string, error) {
-	exePath := filepath.Join(executablePath, "remote.test")
+func BuildGoTests(testsPath string) (string, string, error) {
+	exePath := filepath.Join(utils.ProjectRoot, "remote.test")
```

Comment:
This could lead to issues when running soak tests from other repos. I saw this a while back where we would run from `test-repo/suite` and the soak tests would be looking for an executable in `github.com/smartcontractkit/chainlink-testing-framework@v1/...` and fail. Worth checking running from an outside repo.

Reply:
Nice catch, moved the paths up